### PR TITLE
Add badges showing current filters for tables

### LINKED ISSUE #6: Show the current filters when a filter is applied

### PR DESCRIPTION
**Pull request by trickeydan**
Fixes #6 

image
image

### CODE REVIEW

File: helpdesk/helpdesk/settings.py [hunk 1]
```diff
@@ -94,6 +94,7 @@
 INSTALLED_APPS = [
     "accounts",
     "display",
+    "helpdesk",
```
Comment (PeterJCLaw) [inline helpdesk/helpdesk/settings.py:97]: :raised_eyebrow:

File: helpdesk/helpdesk/templatetags/filter_helpers.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+def _get_value_display_for_field(field: Field, value: Any) -> Any:
+    if isinstance(field, ChoiceField):
+        return _get_value_display_for_choices(field, value)
+
+    return value
+
+
+@register.inclusion_tag("inc/tags/filter_badges.html", takes_context=True)
+def render_filter_badges(context: dict[str, Any]) -> dict[str, Any]:
+    fltr = context["filter"]
```
Comment (PeterJCLaw) [inline helpdesk/helpdesk/templatetags/filter_helpers.py:24]: This feels a bit implicit -- perhaps we should pass this in explicitly?

File: helpdesk/templates/inc/tags/filter_badges.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+            <div class="control">
+                <div class="tags has-addons">
+                    <span class="tag is-dark">{{ label }}</span>
+                    <a class="tag is-link">{{ value }}</a>
+                    <a class="tag is-delete js-modal-trigger" data-target="modal-filter"></a>
+                </div>
+            </div>
+        {% endfor %}
+    </div>
+{% endif %}
```
Comment (PeterJCLaw) [inline helpdesk/templates/inc/tags/filter_badges.html:14]: Missing trailing newline.

**Review by PeterJCLaw — APPROVED** [review]
Generally :thumbsup:

My only concern is that the custom tag using the conventional name `filter` is a bit implicit and we should probably pass the value explicitly at the callsites.

File: helpdesk/helpdesk/settings.py (hunk 1 above)
Comment (trickeydan) [reply helpdesk/helpdesk/settings.py:97]: I'd be interested to know which app you think the template tag belongs in instead?

File: helpdesk/helpdesk/templatetags/filter_helpers.py (hunk 2 above)
Comment (trickeydan) [reply helpdesk/helpdesk/templatetags/filter_helpers.py]: 3f454f1

File: helpdesk/templates/inc/tags/filter_badges.html (hunk 3 above)
Comment (trickeydan) [reply helpdesk/templates/inc/tags/filter_badges.html]: 3f454f1

File: helpdesk/helpdesk/settings.py (hunk 1 above)
Comment (PeterJCLaw) [reply helpdesk/helpdesk/settings.py:97]: It was more that I was surprised this wasn't already here.

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**